python/paddle/fluid/dygraph/learning_rate_scheduler.py

```diff
@@ -626,9 +626,8 @@ def __init__(self,
                 format(learning_rate))
         self.learning_rate = learning_rate
         self.warmup_steps = warmup_steps
-        assert (end_lr > start_lr,
-                "end_lr {} MUST GREATER than start_lr {}".format(end_lr,
-                                                                 start_lr))
+        assert end_lr > start_lr, "end_lr {} MUST GREATER than start_lr {}".format(
```

Comment:
Tiny grammar issue: "MUST be GREATER than"

Reply:
Done.